mypy/checkexpr.py

```diff
@@ -993,6 +993,10 @@ def check_arg(self, caller_type: Type, original_caller_type: Type,
                 return
             messages.incompatible_argument(n, m, callee, original_caller_type,
                                            caller_kind, context)
+        elif ('expr' in self.chk.options.disallow_any and
+                has_any_type(callee_type) and
+                not self.chk.is_stub):
+            messages.call_any_parameter(n, callee.name, callee_type, context)
```

Comment:
Why does this need to be special cased?  Shouldn't these be caught from other places?